templates/node--islandora-rdm-dataset--search-result.html.twig

```diff
@@ -0,0 +1,18 @@
+
+{% set created_date = node.getCreatedTime|format_date('umami_dates') %}
+{{ attach_library('classy/node') }} {# apply classy theme to node}
+                                    
+{# Format Creator (PublicationYear): Title. Publisher. (resourceTypeGeneral). Identifier #}
+<h4>{{ node.field_rdm_creator.0.entity.field_rdm_personal_name.value }} 
+    ({{ node.field_rdm_publication_year.value }}):  
+    <a href="{{url}}"class="node__title-link">{{- label -}}</a>. 
+    {{ node.field_rdm_publisher.0.entity.field_rdm_organization_name.value }}. 
+    {{ node.field_rdm_resource_type_general.value }}. 
+    <a href="https://doi.org/{{node.field_islandora_rdm_identifier.value}}">{{ node.field_islandora_rdm_identifier.value }}</a></h4>
+    
+
+<p>Abstract: {{ rdm_abstract }}</p>
```

Comment:
Make sure this "Abstract: " string is not printed if there is no abstract. Find a Twig function that checks if a variable is empty or not and put it in an if block.

Reply:
Did it in PHP. 
$abs_description = $description->get('field_rdm_description')->getString();
    if ($type == 'Abstract'&& trim($abs_description) != "") {